Bottom: e67796b3b350afda218f4b561dc1b54da04e9814
Top:    8d7444ebee49e19ec6e4b83b39445f96038bae1b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-03 16:50:50 +0100

drc: HACK HACK HACK


---

diff --git a/src/Makefile.am b/src/Makefile.am
index 4943755..353160a 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -285,6 +285,7 @@ libgtk_a_CPPFLAGS = -I./hid/gtk
 LIBGTK_SRCS = \
 	dolists.h \
 	hid/hidint.h \
+	hid/gtk/bga-wizard.c \
 	hid/gtk/gtkhid-main.c \
 	hid/gtk/gtkhid.h \
 	hid/gtk/gui.h \
